sangria/sangria_2.13/src/test/scala/com/linecorp/armeria/server/sangria/SangriaGraphqlSuite.scala

```diff
@@ -83,7 +83,8 @@ class SangriaGraphqlSuite extends FunSuite with ServerSuite {
          }
        """
 
-      val response = executeQuery(server.webClient(), method, query = query, variables = Map("humanId" -> "1002"))
+      val response =
+        executeQuery(server.webClient(), method, query = query, variables = Map("humanId" -> "1002"))
```

Comment:
revert?

Reply:
Let me check. I think that might be automatically fixed by Scalafmt.